<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>wind</class>
	<widget class="QWidget" name="wind">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>853</width>
				<height>657</height>
			</rect>
		</property>
		<property name="font">
			<font>
				<family>SF Pro Rounded</family>
			</font>
		</property>
		<property name="windowTitle">
			<string>Form</string>
		</property>
		<widget class="windRose" name="rose" native="true">
			<property name="geometry">
				<rect>
					<x>89</x>
					<y>49</y>
					<width>571</width>
					<height>391</height>
				</rect>
			</property>
		</widget>
		<widget class="Complication" name="topLeft" native="true">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>155</width>
					<height>114</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<family>SF Pro Rounded</family>
					<pointsize>31</pointsize>
				</font>
			</property>
			<property name="showUnit" stdset="0">
				<bool>false</bool>
			</property>
			<layout class="QGridLayout" name="gridLayout_2">
				<property name="leftMargin">
					<number>10</number>
				</property>
				<property name="topMargin">
					<number>10</number>
				</property>
				<property name="rightMargin">
					<number>0</number>
				</property>
				<property name="bottomMargin">
					<number>0</number>
				</property>
			</layout>
		</widget>
		<widget class="Complication" name="bottomLeft" native="true">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>540</y>
					<width>155</width>
					<height>100</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<family>SF Pro Rounded</family>
					<pointsize>31</pointsize>
				</font>
			</property>
			<property name="showUnit" stdset="0">
				<bool>false</bool>
			</property>
			<layout class="QGridLayout" name="gridLayout">
				<property name="leftMargin">
					<number>10</number>
				</property>
				<property name="topMargin">
					<number>0</number>
				</property>
				<property name="rightMargin">
					<number>10</number>
				</property>
				<property name="bottomMargin">
					<number>0</number>
				</property>
				<property name="spacing">
					<number>0</number>
				</property>
			</layout>
		</widget>
		<widget class="Complication" name="topRight" native="true">
			<property name="geometry">
				<rect>
					<x>690</x>
					<y>0</y>
					<width>155</width>
					<height>114</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<family>SF Pro Rounded</family>
					<pointsize>31</pointsize>
				</font>
			</property>
			<property name="showUnit" stdset="0">
				<bool>false</bool>
			</property>
			<layout class="QGridLayout" name="gridLayout_4">
				<property name="leftMargin">
					<number>10</number>
				</property>
				<property name="topMargin">
					<number>10</number>
				</property>
				<property name="rightMargin">
					<number>0</number>
				</property>
				<property name="bottomMargin">
					<number>0</number>
				</property>
			</layout>
		</widget>
		<widget class="Complication" name="bottomRight" native="true">
			<property name="geometry">
				<rect>
					<x>690</x>
					<y>530</y>
					<width>155</width>
					<height>116</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<family>SF Pro Rounded</family>
					<pointsize>31</pointsize>
				</font>
			</property>
			<property name="showUnit" stdset="0">
				<bool>false</bool>
			</property>
			<layout class="QGridLayout" name="gridLayout_3">
				<property name="leftMargin">
					<number>10</number>
				</property>
				<property name="topMargin">
					<number>10</number>
				</property>
				<property name="rightMargin">
					<number>0</number>
				</property>
				<property name="bottomMargin">
					<number>0</number>
				</property>
			</layout>
		</widget>
	</widget>
	<customwidgets>
		<customwidget>
			<class>Complication</class>
			<extends>QWidget</extends>
			<header>widgets/Complication</header>
			<container>1</container>
			<slots>
				<signal>clicked()</signal>
			</slots>
		</customwidget>
		<customwidget>
			<class>windRose</class>
			<extends>QWidget</extends>
			<header>widgets/WindRose</header>
			<container>1</container>
		</customwidget>
	</customwidgets>
	<resources/>
	<connections/>
</ui>
